Cakefile

```diff
@@ -129,6 +129,11 @@ task 'build:browser', 'rebuild the merged script for inclusion in the browser',
 
 task 'doc:site', 'watch and continually rebuild the documentation for the website', ->
   # Helpers
+  css = fs.readFileSync('./documentation/css/docs.css', 'utf-8') + '\n' +
+        fs.readFileSync('./documentation/css/tomorrow.css', 'utf-8')
```

Comment:
And add another dependency? I was doing this mainly to clean up the folders, to make `v1` into just an “output” folder. With a big CoffeeScript script block, I figured tge goal was source readability over performance.